### ISSUE TITLE
[BUG] Translation error

### ISSUE THREAD

**Issue by epetit91** [prusa3d/Prusa-Firmware-Buddy#3935]
Printer type - [MK4]

Printer firmware version - [6.0.0]

Original or Custom firmware - [Original]

When I change filament, the message displayed in French is not correct.
The current message is (in the file Prusa-Firmware-Buddy_fr.po) :

#: src/gui/dialogs/DialogLoadUnload.cpp:56
msgid "Was filament unload successful?"
msgstr "Le filament a-t-il été bien **chargé** ?"

The correct message must be :

#: src/gui/dialogs/DialogLoadUnload.cpp:56
msgid "Was filament unload successful?"
msgstr "Le filament a-t-il été bien **déchargé** ?"

This translation error was already present in previous versions.

**Comment by 3d-gussner** [comment]
@epetit91 We have on the MK3S this translation `Déchargement du filament réussi?`
https://github.com/prusa3d/Prusa-Firmware/blob/MK3/lang/po/Firmware_fr.po#L2320-L2321

**Comment by epetit91** [comment]
Yes, this Is correct too.Envoyé de mon iPhoneLe 22 avr. 2024 à 08:54, 3d-gussner ***@***.***> a écrit :﻿
@epetit91 We have on the MK3S this translation Déchargement du filament réussi?
https://github.com/prusa3d/Prusa-Firmware/blob/MK3/lang/po/Firmware_fr.po#L2320-L2321
